app/Http/Controllers/Api/SubjectController.php

```diff
@@ -23,7 +23,7 @@ public function show(Repository $repository)
             'name' => 'string',
         ]);
 
-    	return new Subjects($repository->getAll($queries));
+    	return new Subjects($repository->getAll($queries)->first());
```

Comment:
new Subjects() digunakan untuk memanggil resource untuk single record (bukan collection of records). Jadi cuma satu item yang dapat dimasukkan kedalamnya.

$repository->getAll($queries)->first() digunakan untuk mengambil satu record pertama dari collection of records.